internal/replay/replay_http_server.go

```diff
@@ -131,3 +147,109 @@ func (r *ReplayHTTPServer) writedResponse(w http.ResponseWriter, resp *store.Rec
 	_, err := w.Write(resp.Body)
 	return err
 }
+
+func (r *ReplayHTTPServer) proxyWebsocket(w http.ResponseWriter, req *http.Request, chunks []string) {
+	clientConn, err := r.upgradeConnectionToWebsocket(w, req)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Error proxying websocket: %v", err), http.StatusInternalServerError)
+		return
+	}
+	defer clientConn.Close()
+	replayWebsocket(clientConn, chunks)
+}
+
+func (r *ReplayHTTPServer) loadWebsocketChunks(sha string) ([]string, error) {
+	responseFile := filepath.Join(r.recordingDir, sha+".websocket")
+	fmt.Printf("loading websocket response from : %s\n", responseFile)
+	bytes, err := os.ReadFile(responseFile)
+	var chunks = make([]string, 0)
+	if err != nil {
+		fmt.Printf("Error loading websocket response: %v\n", err)
+		return chunks, err
+	}
+
+	i := 0
+	response := string(bytes)
+	for i < len(response) {
+		// Extracts prefix
+		prefix := response[i]
+		if prefix != '>' && prefix != '<' {
+			return nil, fmt.Errorf("invalid message prefix at position %d: expected '>' or '<', got '%c'", i, prefix)
+		}
+		i++ // Move cursor past prefix.
+
+		// Extracts chunk length
+		numStart := i
+		for i < len(response) && unicode.IsDigit(rune(response[i])) {
+			i++
+		}
+		numEnd := i
+		if numStart == numEnd {
+			return nil, fmt.Errorf("missing chunk length after prefix at position %d", numStart-1)
+		}
+		numStr := response[numStart:numEnd]
+		num, err := strconv.Atoi(numStr)
+		if err != nil {
+			return nil, fmt.Errorf("invalid chunk length '%s': %w", numStr, err)
+		}
+
+		// Extracts chunk
+		chunkStart := numEnd
```

Comment:
If the chunk starts with a digit will have a bug? we should probably add something (' ' ?) after the length string?

Reply:
added whitespace between the number and the actual chunk message, this is not a problem for live session (message wraps in '{}'). Agree it could be an issue when used in other sessions